samples/RefreshFlow/AngularApp/.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "typescript.tsdk": "./node_modules/typescript/lib",
+}
```

Comment:
@PinpointTownes We also need to think about whether to include editor specific items including `.vscode` `.xproj` and `launchsettings.json`.